src/amo/components/CollectionAddAddon/index.js

```diff
@@ -44,23 +44,31 @@ export type Props = {|
   filters: CollectionFilters,
 |};
 
+export type DefaultProps = {|
+  clearTimeout: Function,
+  setTimeout: Function,
```

Comment:
Isn't Flow providing a type for these functions?

Reply:
The definitions Flow created for these, as part of the codemod were... weird. I figured we could just go with `Function` as I don't think we need to be particularly worried with type safety with these, but I relented and created better types for them as part of my cleanup.

Reply:
The definitions Flow created for these, as part of the codemod were... weird. I figured we could just go with `Function` as I don't think we need to be particularly worried with type safety with these, but I relented and created better types for them as part of my cleanup.